src/js/stores/AppsStore.js

```diff
@@ -10,6 +11,24 @@ function removeApp(apps, appId) {
   }).value();
 }
 
+function removeTask(tasks, relatedAppId, taskId) {
+  return lazy(tasks).reject(function (task) {
+    return task.id === taskId && task.appId === relatedAppId;
+  }).value();
+}
+
+function processCurrentApp(currentApp) {
```

Comment:
Isn't `processCurrentApp` only doing a "set to empty array if .tasks doesn't exist" (see below) ? Or maybe you mean this can change in the future?